docs/embedding.md

```diff
@@ -179,15 +179,15 @@ By using the `run_function` property, we're able to execute the `download_model`
 
 #### Creating our Modal Class
 
-A modal class allows us to have more fine-grained control over the behaviour of our container. This allows us to separate different portions of the container life cycle
+A modal class allows us to have more fine-grained control over the behaviour of our container. We can separate out the following portions of the container life cycle:
 
 - What to do just once when the container boots up using the  `__enter__` 
 - What should we do when the container is called by other functions by defining  `@method` calls
 - What to do once the container is shut down using the `__exit__` 
 
-More specifically, in our case, we spawn a server once when the container boots up. This state is then preserved in preparation for future requests that other functions might make so that we only incur the cost of initialising a server once. 
+In our case, we spawn a server once when the container boots up. This state is then preserved in preparation for future requests that other functions might make so that we only incur the cost of initialising a server once. 
 
-All the guesswork of managing the life cycle is taken out of the equation for you with Modal by just defining two functions and using a single decorator. Not only so, we can configure our object to run with a specific image and an attached GPU by modifying the `stub.cls` parameters
+All the guesswork of managing the life cycle is taken out of the equation for you with Modal by just defining two functions and using a single decorator. Not only so, we can also configure our stub object to run with a specific image and an attached GPU by modifying the `stub.cls` parameters
```

Comment:
Hmm I think the stub object is slightly different from the `stub.cls` object. In this case, we're trying to get the `stub.cls` object to un with different parameters